Documentation/ceph-upgrade.md

```diff
@@ -471,6 +480,15 @@ It's best to run `ceph config-key dump` again to verify references to
 
 See for more information, see here: https://github.com/rook/rook/issues/9185
 
+### **Rename CephBlockPool device_health_metrics pool when upgrading to Quincy v17**
+In Ceph Quincy (v17), the `device_health_metrics` pool was renamed to `.mgr`. Ceph will perform this
+migration automatically; however, if you use CephBlockPool to customize the configuration of the
+`device_health_metrics` pool, you should update the CephBlockPool to use the new name (`.mgr`) after
+<!-- TODO: after, or before? -->
+updating to Ceph v17. Or you can create a new CephBlockPool for the `.mgr` pool and delete the old
+`device_health_metrics` pool.
+<!-- TODO: @travisn is this right? -->
+
```

Comment:
@travisn could use your eyes here. I'll also test this myself, but you may have better recommendations to start with.

Reply:
Here's the sequence of events I would anticipate.
1. Cluster is configured on v1.8
2. Upgrade Rook to v1.9. The operator is restarted, and all the reconciles settle on the current version of Ceph (e.g. Pacific)
3. When ready, upgrade to Ceph Quincy. The ceph daemons will all be upgraded, and ceph will rename the pool to `.mgr`.
4. Check that the Ceph pool was automatically renamed to `.mgr`.
5. Now the pool CR should be updated: `name: .mgr`. 
 
As long as the rook operator is not restarted between steps 4 and 5 there should be no issue. Step 4 will just cause the pool to be reconciled as `.mgr`, and it will find it already exists. And if the operator does reconcile for some reason including a restart, the worst thing that happens is that the device_health_metrics pool will be created again and remain unused by Ceph. 

So I believe we just need to document those steps 4-5 after upgrading to quincy.